src/jinterface.jl

```diff
@@ -0,0 +1,69 @@
+"""
+Creates a type which incorporates the mesh attributes alongside the necessary values
+to solve finite element problems. It works by taking in a FEniCS mesh, working out
+the necessary values, and then creating an ordering of the internal/boundary nodes
+and applying the DirichletBC (via a function) to the respective nodes.
+"""
+type feMesh
+    n_nodes::Int64
+    n_elements::Int64
+    nodes::Array{Float64, 2}
+    internal_nodes::Array{Int64}
+    boundary_nodes::Array{Int64}
+    elements::Array{Int64, 2}
+    n_internal_nodes::Int64
+    n_boundary_nodes::Int64
+    node_vals::Array{Float64}
+
+    function feMesh(mesh::Mesh,boundaryCondition::Core.Function)
```

Comment:
`Base.Function`?

Reply:
Corrected ( will also check for any other Core / Base discrepancies in the code base